.gitignore

```diff
@@ -363,3 +363,7 @@ TSWLatexianTemp*
 
 # Makeindex log files
 *.lpz
+
+
+.gitignore
```

Comment:
I'm not sure this is useful :)

Reply:
Not working either :/
My bad for forgetting this